navigation2/package.xml

```diff
@@ -22,10 +21,10 @@
   <exec_depend>nav2_dynamic_params</exec_depend>
   <exec_depend>nav2_map_server</exec_depend>
   <exec_depend>nav2_mission_executor</exec_depend>
+  <exec_depend>nav2_motion_primitives</exec_depend>
   <exec_depend>nav2_msgs</exec_depend>
   <exec_depend>nav2_robot</exec_depend>
   <exec_depend>nav2_simple_navigator</exec_depend>
-  <exec_depend>nav2_system_tests</exec_depend>
```

Comment:
why is system tests removed?

Reply:
Not including it in the package xml doesnt have to do with releasing - it will release either way. Its also needed for people to develop on locally to run the test to make sure things are functioning as expected.

Reply:
Ah. Sorry. Should have been clearer. System_tests has all sorts of other dependencies like gazebo and and probably test data at some point. I'm assuming the average person who does apt get install navigation2 doesn't care for those dependencies. They just want to run it.

Reply:
I don't think system tests should be part of the binary package release. Are they normally part of other released packages?

Right now they aren't passing anyway, but when we get them passing and tested, then we could discuss putting them back in if there's a good reason for that.

Reply:
Ok, this works